modules/sdk-coin-ada/src/lib/transactionBuilder.ts

```diff
@@ -10,11 +10,12 @@ import {
   TransactionType,
   UtilsError,
 } from '@bitgo/sdk-core';
-import { Transaction, TransactionInput, TransactionOutput, Withdrawal } from './transaction';
+import { AssetList, Transaction, TransactionInput, TransactionOutput, Withdrawal } from './transaction';
 import { KeyPair } from './keyPair';
 import util from './utils';
 import * as CardanoWasm from '@emurgo/cardano-serialization-lib-nodejs';
 import { BigNum } from '@emurgo/cardano-serialization-lib-nodejs';
+import { Buffer } from 'buffer';
```

Comment:
What is this Buffer import? We can't use native JS Buffer?

Reply:
got auto added I believe, when I was referencing code from stack overflow; removing